efficientdet/anchors.py

```diff
@@ -288,13 +287,13 @@ def _generate_detections_tf(cls_outputs,
 
   image_size = utils.parse_image_size(image_size)
   detections = tf.stack([
-      tf.cast(tf.tile(image_id, tf.shape(top_detection_idx)), image_scale.dtype),
+      tf.cast(tf.tile(image_id, tf.shape(top_detection_idx)), tf.float32),
```

Comment:
@fsx950223 
image_scale.dtype causes some tests failed (such as model_inspect_test.py), so I changed it back to float32.